.circleci/config.yml

```diff
@@ -0,0 +1,188 @@
+version: 2
+jobs:
+    python-3.5: &build
+        docker:
+            - image: circleci/jdk8:0.1.1
+
+        steps:
+            - checkout
+            - restore_cache:
+                key: coala-bears-dep-{{checksum "requirements.txt"}}
+
+            - run:
+                name: dependencies
+                command: |
+                  sudo apt-get update
+                  sudo apt-get install -y software-properties-common python-software-properties curl
+                  sudo add-apt-repository -y ppa:hvr/ghc
+                  sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+                  sudo add-apt-repository -y ppa:avsm/ppa
+                  curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash  -
+                  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

Comment:
why using yarn?

We have another issue about yarn.

I can re-assign that to you, if you explain why it was necessary for Circle 2.0

Reply:
It was pre-installed in the previous ci image
I will check the details, don't remember them well

Reply:
Will also find the yarn issue and see if this can solve that issue